includes/Plugin.php

```diff
@@ -172,14 +174,16 @@ protected function get_bindings() {
 	 */
 	protected function get_shared_instances() {
 		return [
+			Assets::class,
 			Experiments::class,
 			Admin\Meta_Boxes::class,
 			Locale::class,
 			Admin\Activation_Flag::class,
 			Integrations\Site_Kit::class,
 			Analytics::class,
 			Decoder::class,
-			Register_Font::class,
+			Amp_Player_Assets::class,
```

Comment:
Nit: let's call that class `AMP_Story_Player_Assets` to make it clear it's for `<amp-story-player>`

Reply:
Fixed